src/tex/appendix.tex

```diff
@@ -312,15 +311,15 @@ \subsection{Influence of model scale}
 
 \clearpage
 \subsection{Refusal detection}
-
+\Glspl{llm} typically undergo refusal training to prevent harmful or undesirable outputs. As a result, models may decline to answer questions they perceive as potentially adversarial prompts.
```

Comment:
I think this sounds a bit better
```suggestion
\Glspl{llm} typically undergo refusal training to prevent harmful or undesirable outputs. As a result, models may decline to answer questions perceived as potentially adversarial prompts.
```